DataBase/DB-Week-1/week-1.sql

```diff
@@ -0,0 +1,51 @@
+-- Active: 1724566794618@@localhost@3306@myfirstdb
+
+--1.Find out how many tasks are in the task table
+SELECT count(*) AS total_tasks FROM task ;
+ 
+--2.Find out how many tasks in the task table do not have a valid due date
+
+SELECT COUNT(*) AS Invalid_due_date from task where due_date IS NULL;
+
+--3.Find all the tasks that are marked as done
+SELECT t.title, s.name
+FROM task t
+JOIN status s ON t.status_id = s.id
+WHERE s.name = 'Done';
+
+--4.Find all the tasks that are not marked as done
+SELECT t.title, s.name
+from task t
+join status s ON t.status_id = s.id
+WHERE s.name != "Done";
+
+--5.Get all the tasks, sorted with the most recently created first
+select created, title
+from task 
+ORDER BY created DESC;
+
+--6.Get the single most recently created task
+
+SELECT title, created
+from task 
+ORDER BY created DESC LIMIT 1;
+
+--7.Get the title and due date of all tasks where the title or description contains database
+SELECT title, due_date
+FROM task 
+WHERE title LIKE '%database%' or description LIKE '%database%';
+
+--8.Get the title and status (as text) of all tasks
+SELECT title, status.name AS status FROM task JOIN status ON task.status_id = status.id;
+
+--9.Get the name of each status, along with a count of how many tasks have that status
+SELECT s.name, COUNT(t.status_id) as Total_Task
+FROM task t JOIN status s
+on t.status_id = s.id
+GROUP BY t.status_id;
+
+--10.Get the names of all statuses, sorted by the status with most tasks first
```

Comment:
this query is wrong, please take a second look at it

Reply:
I appreciate your review. Could you look over my queries? I edited the line you pointed out a little bit. Is that right now?
Thank you

Reply:
Hello Mentor, Should I merge this branch?

Reply:
Hi @Nusrath-Jahan  I believe the query number 10 still needs some work

Reply:
Thank you for your reply!!
Could you kindly confirm that everything is okay now?

Reply:
Hello jpganz, 
Could you please check the number 10 task ?